app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from typing import List
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: List[Player]) -> int:
+    return sum(player.get_rating() for player in players_list)
+
+
+def elves_concert(elfs_list: List[Elf]) -> None:
+    [elf.play_elf_song() for elf in elfs_list]
```

Comment:
Here's what I were told when did same in knights_fight task:
![NVIDIA_Share_c7YnjcV1c6](https://github.com/mate-academy/py-elves-and-dwarves/assets/13889031/fff11615-142d-4111-b813-8fc7df49fc3c)